view/frontend/layout/default.xml

```diff
@@ -21,7 +21,11 @@
     <body>
         <referenceBlock name="head.additional">
             <block class="Bolt\Boltpay\Block\Js" name="boltjs"    template="Bolt_Boltpay::js/boltjs.phtml"/>
-            <block class="Bolt\Boltpay\Block\Js" name="boltcss"   template="Bolt_Boltpay::css/boltcss.phtml"/>
+            <block class="Bolt\Boltpay\Block\Js" name="boltcss"   template="Bolt_Boltpay::css/boltcss.phtml">
+                <arguments>
+                    <argument name="feature_switches" xsi:type="object">Bolt\Boltpay\ViewModel\FeatureSwitches</argument>
```

Comment:
**Consider**:  Creating a file  ViewModel/FeatureSwitches.php  seems verbose to me. 
Instead of that, I think we can simply create a method isPreventSSOCustomersFromEditingAccountInformation in the JS block and call it in the template view/frontend/templates/css/boltcss.phtml

Reply:
Done